openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java

```diff
@@ -0,0 +1,279 @@
+package io.openbas.rest.settings;
+
+import io.openbas.config.OpenBASConfig;
+import io.openbas.config.OpenBASPrincipal;
+import io.openbas.database.model.Setting;
+import io.openbas.database.model.Theme;
+import io.openbas.database.repository.SettingRepository;
+import io.openbas.injects.opencti.config.OpenCTIConfig;
+import io.openbas.rest.helper.RestBehavior;
+import io.openbas.rest.settings.form.ThemeInput;
+import io.openbas.rest.settings.response.OAuthProvider;
+import io.openbas.rest.settings.response.PlatformSettings;
```

Comment:
There are some unused import in your files.
IntelliJ Idea can handle it for you wirth this shortcut -> Ctrl+Alt+O